test/functional/tests/lazy_writes/cleaning_policy/acp_params.py

```diff
@@ -218,6 +226,174 @@ def test_acp_param_wake_up_time(cache_line_size, cache_mode):
+        core_dirty_blocks_after = int(core.get_dirty_blocks().get_value(Unit.Blocks4096))
+        if core_dirty_blocks_before != core_dirty_blocks_after:
+            TestRun.LOGGER.error(
+                f"Dirty data statistics differs despite the NOP policy was used.\n"
+                f"dirty data blocks before pause: {core_dirty_blocks_before}\n"
+                f"dirty data blocks after pause: {core_dirty_blocks_after}"
+            )
+
+    with TestRun.step("Flush all dirty data."):
+        cache.flush_cache()
+
+    with TestRun.step("Change cleaning policy to ALRU and set parameters."):
+        cache.set_cleaning_policy(CleaningPolicy.alru)
+        default_params = cache.get_flush_parameters_alru()
+
+        params = FlushParametersAlru(
+            wake_up_time=time_to_wait_alru_1st,
+            staleness_time=time_to_wait_alru_1st,
+            flush_max_buffers=blocks_to_flush,
+            activity_threshold=time_to_wait_alru_1st
+        )
+        cache.set_params_alru(params)
+        new_params = cache.get_flush_parameters_alru()
+
+    with TestRun.step("Check if ALRU parameters are configured successfully."):
+        if cache.get_cleaning_policy() != CleaningPolicy.alru:
+            TestRun.fail("ALRU cleaning policy is not set!")
+        if default_params == new_params:
+            TestRun.fail("ALRU parameters are not changed.")
+
+    with TestRun.step("Fill core with dirty data."):
+        dd = (
+            Dd().input("/dev/zero")
+                .output(core.path)
+                .block_size(Size(1, Unit.MiB))
+                .oflag("direct")
+        )
+        dd.run()
+
+    with TestRun.step("Check core statistics."):
+        core_dirty_blocks_before = int(core.get_dirty_blocks().get_value(Unit.Blocks4096))
+
+    with TestRun.step(f"Wait {int((time_to_wait_alru_1st * 1.2).total_seconds())} seconds "
+                      f"and check statistics again."):
+        time.sleep(int((time_to_wait_alru_1st * 1.2).total_seconds()))
+        core_dirty_blocks_after = int(core.get_dirty_blocks().get_value(Unit.Blocks4096))
+        if core_dirty_blocks_before == core_dirty_blocks_after:
+            TestRun.LOGGER.error(
+                f"Dirty data statistics are the same despite the ALRU policy was used.\n"
+                f"dirty data blocks before pause: {core_dirty_blocks_before}\n"
+                f"dirty data blocks after pause: {core_dirty_blocks_after}"
+            )
+        elif core_dirty_blocks_before != (core_dirty_blocks_after + blocks_to_flush):
+            TestRun.LOGGER.error(
+                f"Number of dirty blocks flushed differs from configured in policy.\n"
+                f"configured dirty data blocks to flush: {blocks_to_flush}\n"
+                f"currently dirty data blocks flushed: "
+                f"{core_dirty_blocks_before - core_dirty_blocks_after}"
+            )
+
+    with TestRun.step(f"Wait {int((time_to_wait_alru_2nd * 1.2).total_seconds())} seconds "
```

Comment:
Why is `time_to_wait_alru_2nd` defined and used always with `* 1.2` instead of just being defined as 20% bigger value?